app/react/Documents/components/DocumentSidePanel.js

```diff
@@ -196,7 +197,7 @@ export class DocumentSidePanel extends Component {
             })()}
             <li>
               <TabLink to="metadata" default>
-                <Icon icon="info-circle" />
+                <Icon icon="info-circle" aria-label={t('System', 'Info')} />
                 <span className="tab-link-tooltip">{t('System', 'Info')}</span>
```

Comment:
@elric-wamugu this case is weird, the System text is already on the screen in the span below the Icon, why do we need the aria-label in this case ?
will this be a good usage for [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute#:~:text=Description,in%20a%20space%20delimited%20fashion.) ??

Reply:
The reason the span text does not work is because the `tooltip` class gives it a `display: none` style rule which is then omitted by the screen reader. Putting it on the `Icon` component doesn't work as well because the SVG generated has an `aria-hidden="true"` attribute.